spec/models/post_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject do
+    @user = User.create(name: 'Tom & Jerry', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Best friends',
+                        posts_counter: 12)
+    @post = Post.create(title: 'Physics', text: 'This is not my first post', comments_counter: 10, likes_counter: 10,
+                        author: @user)
+  end
+
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should not be empty' do
+    subject.title = ' '
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should have valid value' do
+    expect(subject.title).to eql 'Physics'
+  end
+
```

Comment:
- [ ] It would be better if we add a test to check for titles not exceeding 250 characters as we have in the project requirements
![Screenshot from 2023-10-23 23-15-32](https://github.com/ANTOINE1128/Blog-App1/assets/46388113/37f337bc-67f0-4a06-a3dd-1276bd2c53f8)